tests/data/policy/agent_policy.rego

```diff
@@ -21,62 +21,50 @@ import rego.v1
 
 policy_version := "0.1.0"
 
+# Defines default rules to be used if not specified in policy file.
 default default_global_rules := {
-	"allowListOnly": false,
+	"allowListOnly": true,
```

Comment:
"allowListOnly" is confusing, since there isn't really an "allow list". There is a list of extensions, but that list does not act only as an allow list, it can be used to set conditions on the signature, irrespective of "allowListOnly"

Reply:
We could change this to "ListedExtensionsOnly"